fhir/data/fhir4_synthea/AaronBrekke_test.json

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
Let's move this file under the doc/ dir, and rename it to Aaron_Brekke_short.json